sljit_src/sljitNativeMIPS_common.c

```diff
@@ -3783,15 +3860,19 @@ static sljit_s32 update_mem_addr(struct sljit_compiler *compiler, sljit_s32 *mem
 }
 
 #if (defined SLJIT_LITTLE_ENDIAN && SLJIT_LITTLE_ENDIAN)
-#define MEM16_IMM_FIRST(memw) IMM((memw) + 1)
-#define MEM16_IMM_SECOND(memw) IMM(memw)
-#define MEMF64_FS_FIRST(freg) FS(freg)
-#define MEMF64_FS_SECOND(freg) (FS(freg) | ((sljit_ins)1 << 11))
+#define IMM_F64_FIRST_LEFT(memw)	IMM((memw) + SSIZE_OF(s32) - 1)
```

Comment:
Why -1 at the end? It overlaps with IMM_F64_FIRST_RIGHT

Reply:
There is no overlap, for example, in a double that is offset by 2 bytes and assuming the address from the aligned buffer  is in `v0` the following will be needed to load `f0` (in legacy mode):

```
	lwl    t9,6(v0)
	lwr    t9,9(v0)
	mtc1   t9,$f0
	lwl    t9,2(v0)
	lwr    t9,5(v0)
	mtc1   t9,$f1
```